### ISSUE TITLE
0.14 win crashes after code evaluation

### ISSUE THREAD

**Issue by Mart-Bogdan** [leksah/leksah#26]
Leksah works. navigation works, build works, when I launch GHCI session everithing fine until I evaluate main function. Leksah crashes just after evaluation (but evaluation is compleated, as I can see result)

GHC: 7.8.3 x86

from HaskellPlatform-2014

Имя события проблемы: APPCRASH
  Имя приложения:  leksah.exe
  Версия приложения:    0.0.0.0
  Timestamp:    5402e0cf
  module name:  libjavascriptcoregtk-3.0-0.dll
  module version:   0.0.0.0
  Timestamp of module with error:   53343a41
  Exception code:   c0000005
  Excelption displacement:  00000000000a5114
  OS Version:   6.1.7601.2.1.0.256.48
  Language code:    1058
  Дополнительные сведения 1:  badc
  Дополнительные сведения 2:  badc3084126d57c907a9a60a70a7f381
  Дополнительные сведения 3:  4b23
  Дополнительные сведения 4:  4b236613ad870287b665c01631c1ce78

![image](https://cloud.githubusercontent.com/assets/775038/4102795/503cec6e-313d-11e4-94e3-894c4e199b98.png)

**Comment by slimbofat** [comment]
Same issue. The result gets printed and then the application crashes.  I am on windows 7.

**Comment by limdingwen** [comment]
Happens to me as well. On Windows 7 64-bit. Whenever I evaluate anything using the debugger pane, suddenly everything turns unresponsive. Sometimes it happens instantly, other times it takes a few seconds to turn unresponsive. Leksah.exe is shown to use an entire CPU core.

No error messages are printed in the console.

![leksahcrash](https://cloud.githubusercontent.com/assets/1744967/14303470/9833438a-fbdd-11e5-8c88-e8cc64822f46.PNG)

**Comment by hamishmack** [comment]
It might be creating the font cache used by WebKitGtk (renders the Output pane).  This can be very slow, but if it works correctly it should only happen once.  Is anything every displayed in the Output pane? Can you try leaving it for a while and see if it comes back to life?  Also if you have [Process Monitor](https://technet.microsoft.com/en-us/sysinternals/processmonitor.aspx) you might be able to see if it is actually doing anything (might be writing font cache info to disk).

**Comment by limdingwen** [comment]
@hamishmack That seems to be the problem. My program is a simple graphics program. When I run main, the header of Out remains unchanged and it works.
![leksa4](https://cloud.githubusercontent.com/assets/1744967/14305132/043a2946-fbec-11e5-9c8d-5d313ef2b0b8.PNG)

However, when I eval "5+2", the header of Out is set and Leksah freezes.
![leksa1](https://cloud.githubusercontent.com/assets/1744967/14305076/6b7d21d6-fbeb-11e5-9547-4ab32b2984ee.PNG)

When it was unresponsive, there was a lot of logging in the console (I couldn't copy them). I waited until the logging has stopped, but the program remained unresponsive. I restarted the program and tried "2+2". Surprisingly, it works, even after restarting.
![leksawor](https://cloud.githubusercontent.com/assets/1744967/14305342/09cedb66-fbee-11e5-9f13-9d523205baca.PNG)
